server/src/main.rs

```diff
@@ -0,0 +1,41 @@
+// WIP: not ready for reviews
+
+pub mod service;
+
+use std::collections::HashMap;
+
+use once_cell::sync::OnceCell;
+use oo7::portal::Keyring;
+use std::future::pending;
```

Comment:
I would run rustfmt from nightly, it should sort the imports here for you

Reply:
Done, thanks!